content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: XXXX'
```

Comment:
+1 to @annajung, make sure the actual title's in place before merging.

Reply:
The title is in place now.